internal/testutil/readme.md

```diff
@@ -0,0 +1,3 @@
+# testutil
+
+some testing functions copied out of github.com/stretchr/testify, which were originally used by atomicfile
```

Comment:
This is fine -- but it's MIT licensed code so we should probably copy and paste the license (which contains the copyright) into this readme file, just in case.

Reply:
done